UnitsNet/CustomCode/Quantity.cs

```diff
@@ -2,42 +2,42 @@
 using System.Collections.Generic;
 using System.Diagnostics.CodeAnalysis;
 using System.Globalization;
-using System.Linq;
 using UnitsNet.Units;
 
 namespace UnitsNet
 {
     public partial class Quantity
     {
-        private static QuantityInfoLookup Default => UnitsNetSetup.Default.QuantityInfoLookup;
+        private static QuantityInfoLookup Quantities => UnitsNetSetup.Default.QuantityInfoLookup;
+        private static UnitParser UnitParser => UnitsNetSetup.Default.UnitParser;
 
         /// <summary>
         /// All enum value names of <see cref="Infos"/>, such as "Length" and "Mass".
         /// </summary>
-        public static string[] Names { get => Default.Names; }
+        public static IReadOnlyCollection<string> Names { get => Quantities.Names; }
```

Comment:
Please note this breaking change: I think there simply is no way around it - we'd have to do it at some point..

I could have probably gotten away with an `IReadOnlyList` but I think this is better: the backing dictionary serves a dual purpose.

Reply:
I ended up keeping the `IReadOnlyCollection<string>` and just changing the `IReadOnlyCollection<QuantityInfo>` into `IReadOnlyList<QuantityInfo>` (fixed in https://github.com/angularsen/UnitsNet/pull/1507/commits/6dc5f0979e1acd6839fde559fd35a9ca0805338f), and the `Names` are now backed by a regular Dictionary, meaning they should come out sorted in whatever order they were provided.

Reply:
> I ended up keeping the `IReadOnlyCollection<string>` and just changing the `IReadOnlyCollection<QuantityInfo>` into `IReadOnlyList<QuantityInfo>` (fixed in [6dc5f09](https://github.com/angularsen/UnitsNet/commit/6dc5f0979e1acd6839fde559fd35a9ca0805338f)), and the `Names` are now backed by a regular Dictionary, meaning they should come out sorted in whatever order they were provided.

Actually this isn't true - the `FrozenDictionary` is going to sort the names alphabetically, not in the order in which they are provided. This is mostly irrelevant for the now, as there is no way to _provide_ anything to the `UnitsNetSetup.Default`..